includes/PoolCounter.php

```diff
@@ -161,7 +161,9 @@ function error( $status ) {
 	 * @param $status Status
 	 */
 	function logError( $status ) {
-		wfDebugLog( 'poolcounter', $status->getWikiText() );
+		$msg = json_encode( $status->getErrorsArray() );
+
+		wfDebugLog( 'poolcounter', $msg);
```

Comment:
Ok, I can change the log message, however part of the original problem was that the error message was being translated - this is part of the getWikiText routine - if we use it we'll get a localized message, that is why I only used the getErrorsArray method